framework/src/modules/http_api/init_steps/setup_servers.js

```diff
@@ -40,6 +41,16 @@ module.exports = ({ components: { logger }, config }) => {
 		expressApp.enable('trust proxy');
 	}
 
+	if (config.apm.enabled) {
+		expressApp.use(
+			swStats.getMiddleware({
+				name: config.apm.options.name,
+				uriPath: config.apm.options.uriPath,
+			}),
+		);
+		logger.info('Enabled Lisk APM');
```

Comment:
Maybe add info about `uri` and `name`?
Also what is `name?

Reply:
its the name of the stats service which user would see if the access the JSON api